### PR TITLE
Upgrade to v3.43.0 of the AWS Terraform Provider

### PR DESCRIPTION
**Pull request by stack72**
Fixes: #1513﻿
Fixes: #1515

### CODE REVIEW

**Comment by stack72** [comment]
@mikhailshilkov this is an expected "breaking" change to the data source. The data source originally had an input for logs when it should only have been an output

File: provider/resources.go [hunk 1]
```diff
@@ -2122,6 +2130,16 @@ func Provider() tfbridge.ProviderInfo {
 				Tok: awsResource(servicecatalogMod, "ProductPortfolioAssociation"),
 			},
 			"aws_servicecatalog_service_action": {Tok: awsResource(servicecatalogMod, "ServiceAction")},
+			"aws_servicecatalog_budget_resource_association": {
+				Tok: awsResource(servicecatalogMod, "BudgetResourceApplication"),
```
Comment (mikhailshilkov) [inline provider/resources.go:2134]: Should it be
```suggestion
				Tok: awsResource(servicecatalogMod, "BudgetResourceAssociation"),
```
?

Comment (stack72) [reply provider/resources.go:2134]: yes, great catch! Fixing now

**Review by mikhailshilkov — APPROVED** [review]
LGTM. Thank you for cleaning up all the examples!

**Pull request merged**